scripts/install_moodle.sh

```diff
@@ -307,8 +317,17 @@ echo_info "Done."
 echo_title "Download and extract Moodle files."
 ###############################################################################
 # Ref.: https://download.moodle.org/releases/supported/
-echo_action "Downloading Moodle 3.8.4 tar file..."
-wget https://download.moodle.org/download.php/direct/stable38/moodle-3.8.4.tgz
+echo_action "Downloading Moodle 3.10.1 tar file..."
```

Comment:
Is there a way to get the moodle version dynamically instead of hard-coding it in the comment?

Reply:
The idea is to be able to install the same version over and over again. I'm afraid that if we get the value dynamically, we will ended up upgrading automatically and this is not what we want here.